lms/templates/courseware/progress.html

```diff
@@ -80,7 +80,7 @@ <h2 class="title">${_("We're working on it...")}</h2>
                     <p class="copy">${_("We're creating your certificate. You can keep working in your courses and a link to it will appear here and on your Dashboard when it is ready.")}</p>
                   </div>
                   <div class="msg-actions"></div>
-                  %elif not is_id_verified or is_unverified:
+                  %elif must_be_id_verified or is_unverified:
```

Comment:
To @adampalay's point: consider what happens here if I'm enrolled in the honor mode of a course and have no valid verification. This statement would seemingly evaluate to True in that case, but it shouldn't. I should be able to see my certificate. Why is `is_unverified` necessary here? It seems like this should be a check for one thing: are you missing a required verification?